v1.10/containership/PRODUCT.yaml

```diff
@@ -0,0 +1,8 @@
+vendor: Containership
+name: Containership Kubernetes Engine (CKE)
+version: v3.0.0
+website_url: https://containership.io
+documentation_url: https://docs.containership.io
+product_logo_url: https://cdn.containership.io/logos/containership/cke.svg
```

Comment:
Could you please link to a version of your logo that says "Containership" underneath, as https://landscape.cncf.io/landscape=non-certified-kubernetes&selected=containership does. Or we can add it for you if you tell us the font to use.

Reply:
updated @dankohn

Reply:
Great.